grr/server/grr_response_server/gui/api_call_router.py

```diff
@@ -725,6 +726,19 @@ def UnscheduleFlow(
     """Unschedules and deletes a previously scheduled flow."""
     raise NotImplementedError()
 
+  @Category("Flows")
+  @ArgsType(api_osquery.ApiGetOsqueryResultsArgs)
+  @ResultBinaryStream()
+  @Http("GET", 
+      "/api/clients/<client_id>/flows/<flow_id>/osquery-results/<format>")
```

Comment:
The previous method aligns arguments, I think this should follow the already established style.